deeplabcut/gui/tabs/train_network.py

```diff
@@ -67,11 +68,43 @@ def _on_engine_change(self, engine: Engine) -> None:
                 layout.show()
             else:
                 layout.hide()
+        self._update_snapshot_selection_widgets_visibility()
+
+    def _update_snapshot_selection_widgets_visibility(self):
+        if self.root.engine == Engine.PYTORCH:
+            self.resume_from_snapshot_label.show()
+            self.snapshot_selection_widget.show()
+            self.detector_snapshot_selection_widget.show()
+        else:
+            self.resume_from_snapshot_label.hide()
+            self.snapshot_selection_widget.hide()
+            self.detector_snapshot_selection_widget.hide()
 
     def _set_page(self):
         self.main_layout.addWidget(_create_label_widget("Attributes", "font:bold"))
         self._generate_layout_attributes()
-        self.main_layout.addWidget(_create_label_widget(""))  # dummy label
+
+        self.resume_from_snapshot_label = _create_label_widget(
+            "Resume training from existing snapshots?",
```

Comment:
I rephrased it to _[...]. When using top-down models, you can [...]_ .
So that the message still makes sense when the user is not in Top-Down mode. Is that OK for you?